packages/document/docs/en/guide/advanced/custom-theme.mdx

```diff
@@ -88,6 +88,8 @@ const Layout = () => (
     top={<div>top</div>}
     /* Bottom of the entire page */
     bottom={<div>bottom</div>}
+    /* Custom MDX components. */
+    components={{ p: props => <p {...props} className="my-4 leading-7" /> }}
```

Comment:
Thanks for your contribution, can you also sync this change to `packages/document/docs/zh/guide/advanced/custom-theme.mdx`

Reply:
I've added the changes, let me know if there's anything else I can do!

Reply:
I think this PR is an opt-in feature without introducing breaking changes. It will be merged after CI passed.